### DISCUSSION TITLE
Welcome to MiniExcel Discussions! (支持中文)

### DISCUSSION THREAD

**Discussion by shps951023** [mini-software/MiniExcel#134] [General]
MiniExcel still many functions that have not been implemented. If there are other ideas, please feel free :D

目前還有很多功能沒實現，假如有其他想法都能提出 😉

目前还有很多功能没实现，假如有其他概念都能提出 😉

**Comment by ArcherCheng** [comment]
非常謝謝您，今天使用了這個工具庫把後端資料轉成Excel Report，使用起來實在太順了，我只有使用 SaveAsByTemplate() 這個轉出功能，真的是輕鬆方便快速，真的再次感謝您

**Reply by shps951023** [reply]
Thanks 😉👍

**Comment by Bruce-ChengHan** [comment]
如果以下問題不適合提問請讓我知道，謝謝

我在mac使用dotnet core 3.1 + miniExcel 0.19.0
dotnet build會回報以下錯誤，能否提供線索協助我解決問題
error NU3037: 來自來源 'https://api.nuget.org/v3/index.json' 的套件 'ExcelNumberFormat 1.1.0': 存放庫主要簽章 已過了有效期間。

**Reply by shps951023** [reply]
@Bruce-ChengHan 
https://github.com/shps951023/MiniExcel/issues/302 解決此問題

**Reply by Bruce-ChengHan** [reply]
感謝回覆

雖然我在執行相關語法還是會回報一樣的錯誤
但後續增加了global.json，並指定了dotnet core版本
在不是dotnet core 5的情況下成功了
<img width="281" alt="截圖 2021-11-17 下午2 17 37" src="https://user-images.githubusercontent.com/84494249/142145237-48124fb4-0e5a-41b3-a7ff-003955594884.png">

**Reply by shps951023** [reply]
NuGet 官方回應了 : https://github.com/NuGet/NuGetGallery/issues/8918
主要是之前 NuGet client 過期導致，新的 dotnet 版本不會有這問題

**Reply by Bruce-ChengHan** [reply]
感謝你的協助與熱情
讓我有機會更了解這部分的內容

**Comment by SHUNCSHARP** [comment]
您好，怎么使用MINIExcel，将DataTable中的数据追加到已有模板Excel的某个sheet中，而不是新建sheet？关于MINIExcel中的SaveAsByTemplate方法和SaveAs方法，经测试，是新建sheet了，因此使用这2种方法时，把已有的模板sheet替换了，这点有点糟糕。期望有乐意回答的developer，当然，肯定会有这样好的方法，可在MINIExcel来实现该追加功能，即如同MINIExcel.append（string path, string templatePath, object value）新功能的出现。

**Reply by shps951023** [reply]
@SHUNCSHARP 可以帮忙建立个issue，后续我想一下算法

**Comment by CodeBearMK** [comment]
不知道有沒有可以設定全局字體大小?
不用到指定Row或Column，全部套用固定字體大小的功能?

**Comment by Hunglmc** [comment]
I need to repeat the table and note values ​​in the sheet with the sample template. Can you share with me how to do it?
I want make like this
image
![image](https://github.com/mini-software/MiniExcel/assets/39206819/9b947b91-62dc-48f6-a4d6-bf90d00a6789)

**Comment by wingdragon** [comment]
I would suggest adding the following functions:
1) set the alignment of a column
2) set the styles (e.g. font, color, background color) of a cell
3) auto adjustment of column widths
4) apply different DynamicColumns to multiple worksheets
